elements/rh-subnav/rh-subnav.ts

```diff
@@ -94,29 +91,22 @@ export class RhSubnav extends LitElement {
     return this.#allLinks.at(-1) as HTMLAnchorElement;
   }
 
-  get #activeItem(): HTMLAnchorElement {
-    const activeLink = this.#allLinks.find(link => link.matches('[active]'));
-    return activeLink ?? this.#firstLink;
-  }
-
   connectedCallback() {
     super.connectedCallback();
     RhSubnav.instances.add(this);
-    this.#mo.observe(this, { subtree: true, attributes: true, attributeFilter: ['active'] });
   }
 
   disconnectedCallback() {
     super.disconnectedCallback();
     RhSubnav.instances.delete(this);
-    this.#mo.disconnect();
   }
 
   render() {
     const { scrollIconSet, scrollIconLeft, scrollIconRight } = this.constructor as typeof RhSubnav;
     const { showScrollButtons } = this.#overflow;
     const { on = '' } = this;
     return html`
-      <nav part="container" class="${classMap({ [on]: !!on })}">${!showScrollButtons ? '' : html`
+      <nav part="container" aria-label="${this.accessibleLabel}" class="${classMap({ [on]: !!on })}">${!showScrollButtons ? '' : html`
```

Comment:
Don't output the `aria-label` attribute if `accessibleLabel` is empty.

Reply:
`accessibleLabel` as coded by default is set to `subnavigation` so it should never be empty.  

We prolly need additional documentation around this now that I think about it.  A `rh-subnav` should not be used alone on a page unless otherwise `accessible-label` is set to something like `Main`.  

I guess the question is: what is the least harm, using it with multiple navigations on the page as intended and not applying a label, or having to set the label if it is the sole navigation on the page?

https://github.com/RedHat-UX/red-hat-design-system/blob/6d3d8a3e38ef18d7879826114b6f6645f6b94064/elements/rh-subnav/rh-subnav.ts#L67

Reply:
Least harm: set the label.